src/Mount.cpp

```diff
@@ -86,6 +86,10 @@ Mount::Mount(LcdMenu *lcdMenu)
 
 void Mount::initializeVariables()
 {
+    // We are now defaulting to northern hemisphere at 45deg. Switching is now supported
+    // at runtime when the Latitude is received via Meade command.
+    inNorthernHemisphere = true;  // Since we set latitude to +45 below.
```

Comment:
we should allow changing the default (for standalone builds in SH)

Reply:
Done.